apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -291,6 +291,12 @@ export const ModerationNotification = (props) => {
               </ul>
             </div>
           </div>
+          <div className="col-12 d-md-none">
```

Comment:
not sure if there's a better way than just duplicating the code from line 263, pulling it into a new component somehow also seemed a bit unnecessary. Or maybe there even is just a better way of fixing this issue?

Reply:
yes, same here, the expected behaviour was not clear from the github issue and screenshots.

Reply:
the three dots dropdown in the images should always be on the top right of the reported comment, instead it moves around on smaller screens

Reply:
@hom3mad3 I couldn't find any mobile screens for this (I think we generally don't have a lot of mobile screens for a+, right @mcastro-lqd ? )

Reply:
@goapunk @hom3mad3 We do have mobile: That's because it was made for kosmo which is here: https://zpl.io/llvlYye

Reply:
@mcastro-lqd ah, thanks! @hom3mad3 my changes should (hopefully) match the linked design